apcd-cms/src/apps/admin_exception/templates/view_admin_exception_modal.html

```diff
@@ -15,39 +15,43 @@ <h4 class="modal-title">Exception Details</h4>
-                        <dd class="c-data-list__value">{{r.status}}</dd>
-                        <dt class="c-data-list__key">Outcome</dt>
-                        <dd class="c-data-list__value">{{r.outcome}}</dd> 
-                        <dt class="c-data-list__key">Organization</dt>
-                        <dd class="c-data-list__value">{{r.org_name}}</dd> 
-                        <dt class="c-data-list__key">Requestor Email</dt>
-                        <dd class="c-data-list__value">{{r.requestor_email}}</dd>
-                        <dt class="c-data-list__key">Requestor</dt>
-                        <dd class="c-data-list__value">{{r.requestor_name}}</dd>                      
-                        <dt class="c-data-list__key">File Type</dt>                
-                        <dd class="c-data-list__value">{{r.data_file_name}}</dd>
-                        <dt class="c-data-list__key">Field Number</dt>                
-                        <dd class="c-data-list__value">{{r.field_number}}</dd>
-                        <dt class="c-data-list__key">Required Threshold</dt>                
-                        <dd class="c-data-list__value">{{r.required_threshold}}</dd>
-                        <dt class="c-data-list__key">Requested Threshold</dt>                
-                        <dd class="c-data-list__value">{{r.requested_threshold}}</dd>
-                        <dt class="c-data-list__key">Approved Threshold</dt>                
-                        <dd class="c-data-list__value">{{r.approved_threshold}}</dd>
-                        <dt class="c-data-list__key">Requested Expiration Date</dt>                
-                        <dd class="c-data-list__value">{{r.requested_expiration_date}}</dd>
-                        <dt class="c-data-list__key">Approved Expiration Date</dt>                
-                        <dd class="c-data-list__value">{{r.approved_expiration_date}}</dd>
-                        <dt class="c-data-list__key">Explanation Justification</dt>
-                        <dd class="c-data-list__value">{{r.explanation_justification}}</dd>
-                    </dl> 
-                    <hr>
-                </div>
+                      <dt class="c-data-list__key">Created</dt>
+                      <dd class="c-data-list__value">{{r.created_at}}</dd>
+                      <dt class="c-data-list__key">Organization</dt>
+                      <dd class="c-data-list__value">{{r.org_name}}</dd>
+                      <dt class="c-data-list__key">Requestor</dt>
+                      <dd class="c-data-list__value">{{r.requestor_name}}</dd>
+                      <dt class="c-data-list__key">Requestor Email</dt>
+                      <dd class="c-data-list__value">{{r.requestor_email}}</dd>
+                      <dt class="c-data-list__key">Exception Type</dt>
+                      <dd class="c-data-list__value">{{r.exception_type}}</dd>  
```

Comment:
```suggestion
                      <dd class="c-data-list__value">{{r.request_type}}</dd>  
```
Aligning this var with that in the view.